config/default.yml

```diff
@@ -1,3 +1,8 @@
+Minitest/DuplicatedTestRun:
+  Description: 'This cop detects when a test class has test methods, and its parent also has tests methods causing the parents tests to run them twice.'
+  Enabled: pending
+  VersionAdded: '<<next>>'
```

Comment:
Can you move it to the position in ASCII order?

Reply:
I added a test checking that the cops are sorted in the following PR https://github.com/rubocop/rubocop-minitest/pull/165